Resources/Prototypes/Entities/Arcadis_station/Clothing/Inner/Syndicate/Jumpskirts/syndicatejumpskirts.yml

```
@@ -0,0 +1,142 @@
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpskirtAtmosSyndicate
+  name: syndicate atmospheric technician jumpskirt
+  description: A jumpskirt perfect for letting everyone know you're pumping plasma into distro.
+  components:
+  - type: Sprite
+    sprite: _Arc/Clothing/Syndicate_jobs/Jumpskirt/atmosf.rsi
+  - type: Clothing
+    sprite: _Arc/Clothing/Syndicate_jobs/Jumpskirt/atmosf.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpskirtBartenderSyndicate
+  name: syndicate bartender's uniform
+  description: Wear this while serving syndicate bombs of either kind.
+  components:
+  - type: Sprite
+    sprite: _Arc/Clothing/Syndicate_jobs/Jumpskirt/bartender.rsi
+  - type: Clothing
+    sprite: _Arc/Clothing/Syndicate_jobs/Jumpskirt/bartender.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpskirtChemistSyndicate
+  name: syndicate chemistry jumpskirt
+  description: MORE METH, yells the passenger as the chemists work tirelessly.
```

```suggestion
  description: "MORE METH!" yells the passenger as the chemists work tirelessly.
```
missed this one in my first review